projects/graph/graph.py

```diff
@@ -16,42 +15,66 @@ def add_vertex(self, vertex):
         """
         Add a vertex to the graph.
         """
-        pass  # TODO
+        self.vertices[vertex] = set()
```

Comment:
You should add some conditioning here to prevent existing vertices in the Graph from being added again.